net/dns-over-https/Makefile

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2021 Martin Schneider <martschneider@google.com>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dns-over-https
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/m13253/dns-over-https
+PKG_SOURCE_DATE:=2021-09-29
+PKG_SOURCE_VERSION:=b338c7ae52bed0d1a5e85e30c76d023efb46580d
+PKG_MIRROR_HASH:=ab15dbefac0c9bb0d5763749b532284568002d5b4796ff61b8ca3b51cae771e7
+
+PKG_MAINTAINER:=Martin Schneider <martschneider@google.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/m13253/dns-over-https
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/dns-over-https
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=IP Addresses and Names
+  TITLE:=Client and server software to query DNS over HTTPS
+  URL:=https://github.com/m13253/dns-over-https
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/dns-over-https/description
+  Client and server software to query DNS over HTTPS, using Google DNS-over-HTTPS protocol and IETF DNS-over-HTTPS (RFC 8484).
+endef
+
+define Package/dns-over-https/install
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(CP) $(GO_PKG_BUILD_BIN_DIR)/doh-client $(1)/usr/sbin/
+	$(CP) $(GO_PKG_BUILD_BIN_DIR)/doh-server $(1)/usr/sbin/
```

Comment:
If you set `GO_PKG_INSTALL_BIN_PATH:=/usr/sbin` (and if `doh-client` and `doh-server` are the only binaries built), then this

```make
$(call GoPackage/Package/Install/Bin,$(1))
```

will work. Or you can do

```make
$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
$(INSTALL_DIR) $(1)/usr/sbin
$(CP) $(PKG_INSTALL_DIR)/usr/bin/doh-client $(1)/usr/sbin/
$(CP) $(PKG_INSTALL_DIR)/usr/bin/doh-server $(1)/usr/sbin/
```

Or just

```make
$(INSTALL_DIR) $(1)/usr/sbin
$(CP) $(GO_PKG_BUILD_BIN_DIR)/doh-client $(1)/usr/sbin/
$(CP) $(GO_PKG_BUILD_BIN_DIR)/doh-server $(1)/usr/sbin/
```